app/records/models.py

```diff
@@ -115,108 +128,94 @@ def reference_number(self) -> str:
         """Returns the api value of the attr if found, empty str otherwise."""
-        try:
-            return self.get("@template.details.summaryTitle")
-        except ValueExtractionError:
-            # value from other places
-            try:
-                return self.get("summary.title")
-            except ValueExtractionError:
-                pass
-        return ""
+        if details_summary_title := self.get("summaryTitle", ""):
+            return details_summary_title
+        return self.get("summary.title", "")
 
     @cached_property
     def date_covering(self) -> str:
         """Returns the api value of the attr if found, empty str otherwise."""
-        return self.template.get("dateCovering", "")
+        return self.get("dateCovering", "")
 
     @cached_property
     def creator(self) -> list[str]:
         """Returns the api value of the attr if found, empty list otherwise."""
-        return self.template.get("creator", [])
+        return self.get("creator", [])
 
     @cached_property
     def dimensions(self) -> str:
         """Returns the api value of the attr if found, empty str otherwise."""
-        return self.template.get("dimensions", "")
+        return self.get("dimensions", "")
 
     @cached_property
     def former_department_reference(self) -> str:
         """Returns the api value of the attr if found, empty str otherwise."""
-        return self.template.get("formerDepartmentReference", "")
+        return self.get("formerDepartmentReference", "")
 
     @cached_property
     def former_pro_reference(self) -> str:
         """Returns the api value of the attr if found, empty str otherwise."""
-        return self.template.get("formerProReference", "")
+        return self.get("formerProReference", "")
 
     @cached_property
     def language(self) -> str:
         """Returns the api value of the attr if found, empty str otherwise."""
-        return self.template.get("language", "")
+        return self.get("language", "")
 
     @cached_property
     def legal_status(self) -> str:
         """Returns the api value of the attr if found, empty str otherwise."""
-        return self.template.get("legalStatus", "")
+        return self.get("legalStatus", "")
 
     @cached_property
     def level(self) -> str:
         """Returns the api value of the attr if found, empty str otherwise."""
-        return self.get("@template.details.level.value", "")
+        return self.get("level.value", "")
 
     @cached_property
     def level_code(self) -> int | None:
         """Returns the api value of the attr if found, None otherwise."""
-        try:
-            return self.get("@template.details.level.code")
-        except ValueExtractionError:
-            # check other places
-            try:
-                return self.get("level.code")
-            except ValueExtractionError:
-                pass
-        return None
+        return self.get("level.code", "")
```

Comment:
```suggestion
        return self.get("level.code", None)
```
api level code is `int`, record attribute defaults to `None`

Reply:
It shouldn't really matter - both are falsy values